audformat/core/utils.py

```diff
@@ -1143,6 +1143,106 @@ def set_index_dtypes(
     return index
 
 
+def symmetric_difference(
+        objs: typing.Sequence[typing.Union[pd.Index]],
+) -> pd.Index:
+    r"""Symmetric difference of index objects.
+    If all index objects are conform to
+    :ref:`table specifications <data-tables:Tables>`
+    and at least one object is segmented,
+    the output is a segmented index.
+    Otherwise,
+    requires that levels and dtypes
+    of all objects match,
+    see :func:`audformat.utils.is_index_alike`.
+    When the symmetric difference of a
+    :class:`pandas.Index`
+    with a single-level
+    :class:`pandas.MultiIndex`,
+    is calculated,
+    the result is a
+    :class:`pandas.Index`.
+    Args:
```

Comment:
```suggestion

    Args:
```

Reply:
Weird it removed those blanks when I copied the function from the other branch. I fixed this and the other spots in the last commit.